src/helpers/cursor.ts

```diff
@@ -9,6 +9,7 @@ const DOWN = "B";
 const RIGHT = "C";
 const UP = "A";
 const CLEAR_DOWN = "0J";
+const CLEAR_RIGHT = "0P";
```

Comment:
use `0k` for clear until end of current line, please.

Reply:
of course @disizali dear.